pkgs/build-support/php/hooks/default.nix

```diff
@@ -1,9 +1,11 @@
-{ makeSetupHook
+{ lib
+, makeSetupHook
 , jq
 , moreutils
 , makeBinaryWrapper
 , php
 , cacert
+, buildPackages
```

Comment:
I think `diffutils` is enough, what's the reason to use `buildPackages` here ?

Reply:
Explained on Matrix that `buildPackages` need to be used here. Because:

> usually the nativeBuildInputs and buildInputs do some fancy splicing magic to select the correct version of a package
> but since we are not using stdenv.mkDerivation and nativeBuildInputs here, it should be specified from buildPackages